projects/mci/src/mci_broadcast.c

```diff
@@ -80,6 +80,27 @@ static void prv_broadcast_dsp_temp(MotorControllerBroadcastStorage *storage) {
                                (uint32_t)measurements[RIGHT_MOTOR_CONTROLLER].dsp_temp_c);
 }
 
+static void prv_temperature_fan_toggle(MotorControllerBroadcastStorage *storage) {
```

Comment:
Also, 'toggle' implies that the function is guaranteed to switch the state. Something like prv_update_fan_state might be better